.pyup.yml

```diff
@@ -6,11 +7,11 @@ update: all
 # configure dependency pinning globally
 # default: True
 # allowed: True, False
-pin: True
+pin: true
```

Comment:
Have you checked that pyup supports it like this?

Reply:
@webknjaz In fact I just raised https://github.com/pyupio/pyup/issues/346 after testing it on my own repository. I guess that until they adress it, I will still have to keep one exclude.

Reply:
What I used was copy-pasted from the official example. So yeah. Such things shouldn't be blindly changed. Also, I can see why they might've chosen title-case: it matches Python literals...